iam.tf

```diff
@@ -33,7 +33,10 @@ resource "aws_iam_role_policy" "int_tableau" {
         "arn:aws:ssm:eu-west-2:*:parameter/int_tableau_hostname",
         "arn:aws:ssm:eu-west-2:*:parameter/tableau_server_username",
         "arn:aws:ssm:eu-west-2:*:parameter/tableau_server_password",
-        "arn:aws:ssm:eu-west-2:*:parameter/tableau_int_s3_prefix"
+        "arn:aws:ssm:eu-west-2:*:parameter/tableau_int_s3_prefix",
+        "arn:aws:ssm:eu-west-2:*:parameter/tableau_linux_ssh_private_key",
+        "arn:aws:ssm:eu-west-2:*:parameter/tableau_linux_ssh_public_key",
+        "arn:aws:ssm:eu-west-2:*:parameter/DATA_ARCHIVE_TAB_INT_BACKUP_URL"
```

Comment:
Why is this variable in caps?

Reply:
That's how I named the parameter store entry whilst we were testing pulling a backup from S3 (it just matches the env var that is set). The parameter store entry can be renamed and that line updated if it's an issue